src/mock/utils.go

```diff
@@ -27,8 +27,17 @@ func WriteError(
 ) {
 	w.WriteHeader(httpStatus)
 
-	w.Write(MustMarshal(github.ErrorResponse{
+	w.Write(MustMarshal(gitHubErrorReponse{
 		Message: msg,
 		Errors:  errors,
 	}))
 }
+
+// A struct with the fields that we need from github.ErrorResponse.
```

Comment:
Hi @stoneman , I'm sorry it has taken quite a while for me to get around reviewing this.

I will have a look at this later today. Thanks for contributing!